deny.toml

```diff
@@ -0,0 +1,237 @@
+# This template contains all of the possible sections and their default values
```

Comment:
I think it'll be clearer to have a small config that only changes what matters instead of including the entire template